paasta_tools/kubernetes_tools.py

```diff
@@ -473,6 +473,14 @@ def get_kubernetes_environment(self) -> Sequence[V1EnvVar]:
                     ),
                 ),
             ),
+            V1EnvVar(
+                name='POD_NAME',
```

Comment:
Would be good to add a comment on why this is required:  operator specification requires POD_NAME to be passed during deployment. etc., so that environment variable is not renamed in future (which will break lot of things).

Reply:
right, I'll comment on that.